packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-policy-group-form.vue

```diff
@@ -8,108 +8,77 @@
 					placeholder="Policy bounds"
 					@focusout="emit('update-self', config)"
 				/>
-				<h6 v-else>{{ props.config.name }}</h6>
+				<h6 v-else>{{ config.name }}</h6>
 				<i
 					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
 					:style="'cursor: pointer'"
 					@click="onEdit"
 				/>
 			</div>
 			<div>
-				<label for="active">Active</label>
-				<InputSwitch v-model="config.isActive" @change="emit('update-self', config)" />
-			</div>
-			<div>
-				<i
-					class="trash-button pi pi-trash"
-					:style="'cursor: pointer'"
-					@click="emit('delete-self')"
+				<label for="active">Optimize</label>
+				<InputSwitch
+					v-model="config.isActive"
+					:disabled="config.isDisabled"
+					@change="emit('update-self', config)"
 				/>
 			</div>
+			<!--			<div>-->
+			<!--				<i-->
+			<!--					class="trash-button pi pi-trash"-->
+			<!--					:style="'cursor: pointer'"-->
+			<!--					@click="emit('delete-self')"-->
+			<!--				/>-->
+			<!--			</div>-->
 		</div>
-		<div class="input-row">
-			<div class="label-and-input">
-				<label for="parameter">Parameter</label>
-				<Dropdown
-					class="p-inputtext-sm"
-					:options="props.parameterOptions"
-					v-model="config.parameter"
-					placeholder="Select"
-					@update:model-value="emit('update-self', config)"
-				/>
-			</div>
-			<div class="label-and-input">
-				<label for="initial-guess">Initial guess</label>
-				<tera-input
-					type="number"
-					v-model="config.initialGuess"
-					@update:model-value="emit('update-self', config)"
-				/>
-			</div>
-			<div class="label-and-input"></div>
+
+		<div v-if="dynamicInterventions.length > 0">
+			<tera-dynamic-intervention-policy-group
```

Comment:
We are not correctly passing update events through with these new components which means our state will not properly get updated - let me know if you want to chat about this

https://github.com/DARPA-ASKEM/terarium/assets/17088680/2a72fbe7-cffc-4c79-9443-31727439a3fe

Reply:
https://github.com/DARPA-ASKEM/terarium/issues/4079